tests/utils.py

```diff
@@ -286,3 +303,10 @@ def inner(*args, **kwarg):
 
 def _is_float(value):
     return isinstance(value, (float, np.floating))
+
+
+def format_float(value):
+    if np.isnan(value):
+        return "NaN"
```

Comment:
Just wondering why couldn't we extend the existing function (https://github.com/BayesWitnesses/m2cgen/blob/master/m2cgen/interpreters/utils.py#L29) with this logic?

Reply:
Output of this two functions in theory is different. https://github.com/BayesWitnesses/m2cgen/blob/master/m2cgen/interpreters/utils.py#L29 is used for model code generation and should be language-specific, for example, returning Float.NaN is case of java. Yet nan seems to never be present in actual model, so no need to modify that function at least for now.

Here format_float generates representation for command-line string parsers, and "NaN" works for all supported languages.

Reply:
Thanks for clarifying!

Reply:
> Yet nan seems to never be present in actual model, so no need to modify that function at least for now.

I agree with this statement! But also we should avoid code duplication. It is possible we will need to modify `np.format_float_positional` arguments in the near future quite often, because it was added to the codebase several days ago. With the current design we have to do it in two places.

Please try to reuse the code of `format_float` from `interpreters/utils.py` with decorator or simply 
```
if np.isnan(value):
    return "NaN"
else:
    return m2cgen.interpreters.utils.format_float(value)
```

Also, I guess it will be better to have a new unique name for this function. Something like `format_float_with_nan`.